include/seqan3/core/add_enum_bitwise_operators.hpp

```diff
@@ -34,7 +34,8 @@ namespace seqan3
 template <typename t>
 constexpr bool add_enum_bitwise_operators = false;
 
-/*!\name Binary operators for scoped enums
+//!\cond DEV
+/*!\interface enum_bitwise_operators
```

Comment:
It could be that it works like this, but here is what I thought:

The `\interface`  should not be here, it should be more like in, e.g., the fm_index_concept https://github.com/seqan/seqan3/blob/971b3e41524892431fa686981c52975709e0b7f4/include/seqan3/search/fm_index/concept.hpp#L144

You do the `\interface` and then you can document stuff that belongs to the interface, you can also use `\fn` to write function signatures without actually defining cpp functions: https://www.doxygen.nl/manual/commands.html#cmdfn
I think we use `\fn` somewhere :)

Also, the interface documentation should then be more verbose, e.g. what does `&` mean?

Reply:
I am not sure, I know what you mean. 
So, I should move the `\interface` to line 22?
And then explain what every operator does? Should this then be part of DEV too or not?
Plus, does it really make sense to explain every operator, I would assume the reference to the typical binary operators is enough.

`seqan3::scoring_scheme_for` uses `\fn`, I assume you want every operator explained with an `\fn`?

Reply:
You need to write something like the concepts:
```
/!*\interface seqan3::bitwise_enum_operators

  \brief blablaba
  Does ....
  \fn operator&
*/
```

One sentence for each operator is enough, but all need to be documented since we want to tag the other classes with this interface.
The definition of the operators goes into DEV (just like you already did, but the interface is public).